README.md

```diff
@@ -22,3 +22,13 @@ For safety reasons, this repository cannot be pushed to directly. To make change
 The following link should be used for access to the approved-books list JSON file:
 
 https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-books.json
+
+## Run Tests
+
+```console
+
+$ cd content-manager-approved-books
+$ npm install 
+$ npx ava test.js
```

Comment:
Should we document this as `npm run test` since there's a [test script defined here](https://github.com/openstax/content-manager-approved-books/blob/2a04ea858f31d414b42447afa79bf80d3c525a60/package.json#L5)?

Reply:
ah, yes thank you. - I didn't know how that all worked.